.gitignore

```diff
@@ -1,4 +1,477 @@
-# Ignore any files in /bin and /obj Folders
-**/bin/*
-**/obj/*
-.vs
\ No newline at end of file
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Let's get this into the master branch for your next attempt, then you don't have to keep adding it to each branch.